packages/snap-preact/components/src/components/Templates/AutocompleteTemplate/AutocompleteTemplate.tsx

```diff
@@ -756,7 +710,7 @@ export const AutocompleteTemplate = observer((properties: AutocompleteTemplatePr
 			return <Banner {...subProps.banner} content={merchandising.content} type={ContentType.LEFT} name={'left'} />;
 		}
 
-		if (module == 'Button.see-more' && showResults) {
+		if (module == 'Button.see-more' && showResults && search?.query?.string && results.length > 0) {
```

Comment:
I'm wondering if we should lowercase these layout module strings - I say this because the CCP selectors are lowercase... We could support both (in both places)... 🤷🏻 

Thoughts?